src/app/parser/open-api/param-types.ts

```diff
@@ -288,6 +288,33 @@ export function getEmptySchema(): Schema {
   };
 }
 
+/**
+ * Process a Schema and all its children to determine if a relaxed type tag is required
+ * @param schema 
+ * @param schemaStore 
+ * @returns 
+ */
+export function isRelaxedTypeTagRequiredForSchema(schema: Schema, schemaStore: ParsedSchemaStore): boolean {
+  if (schemaIsTypeRef(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForRefTypeSchema(schema, schemaStore);
+  } else if (schemaIsTypeObject(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForObjectTypeSchemaDeep(schema, schemaStore);
+  } else if (schemaIsTypeScalar(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForScalarTypeSchema(schema);
+  } else if (schemaIsTypeCustomType(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForCustomTypeSchema(schema, schemaStore);
+  } else if (schemaIsTypeArray(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForArrayTypeSchemaDeep(schema, schemaStore);
+  } else if (schemaIsTypeOneOf(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForOneOfTypeSchema(schema);
+  } else if (schemaIsTypeAnyOf(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForAnyOfTypeSchema(schema);
+  } else if (schemaIsTypeAllOf(schema)) {
+    schema._$requiresRelaxedTypeTag = isRelaxedTypeTagRequiredForAllOfTypeSchema(schema);
+  }
+  return schema._$requiresRelaxedTypeTag ?? false;
```

Comment:
Return `true` as default

Reply:
Resolved in [7841a48](https://github.com/hasura/ndc-open-api-lambda/pull/74/commits/7841a4866e1c874ec674a49a3c96a3da8989593c)